entity-framework/core/providers/sqlite/limitations.md

```diff
@@ -17,6 +17,25 @@ The common relational library (shared by Entity Framework relational database pr
 * Sequences
 * Computed columns
 
+## Query limitations
```

Comment:
Are you going to add a breaking change entry for this too? I think that was what we wanted :smile: cc @ajcvickers

Reply:
I was, but I think most of it throws in 2.2, so it may not be as severe as we initially thought

Reply:
Okay with me.